src/OrleansRuntime/Placement/PreferLocalPlacementDirector.cs

```diff
@@ -15,6 +15,10 @@ internal class PreferLocalPlacementDirector : RandomPlacementDirector
         internal override Task<PlacementResult> 
             OnAddActivation(PlacementStrategy strategy, GrainId grain, IPlacementContext context)
         {
+            // if local silo is not active, revert to random placement
+            if (context.LocalSiloStatus != SiloStatus.Active)
```

Comment:
Yes.

IPlacementContext is an internal interface - why would the changes be "breaking"?